config/boot.rb

```diff
@@ -1,6 +1,4 @@
-# frozen_string_literal: true
```

Comment:
It seems like you remove these magic comments and switch from double to single quotes in a few places - what's the reason for these changes?

Reply:
What I do for a rails update is run `rake app:update` and allow it to overwrite all the files with the new versions. Then I go through and look at the changes and revert as necessary. I go back and forth on what to do with these kind of changes. The Rails templates it spits out use single quotes and don't include the `frozen_string_literal` lines. 

On one hand, changing it matches our style guide (well, we don't really enforce rubocop anymore). On the other hand, it's extra noise for future Rails upgrades. This file is identical between 5.2 and 6.0 so if we had not rubocopped it before, there would be no change to the file.

I could be convinced either way. This was just where I was leaning when I did the commit.